v3/ploopyco/trackball/trackball_adept.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "Ploopy Adept Trackball",
```

Comment:
The Ploopy Adept Trackball uses the Madryomys PCB. I imagine the ploopy developers would prefer the Adept name to appear in Via to reduce confusion.

Reply:
Could you please rename the file to Madromys? And perhaps include the it in the name too?

Just because all boards reflect the QMK tree and naming. 

https://www.caniusevia.com/docs/supported_keyboards

Also because it shows up here and for quick searching purposes.

Reply:
How about

file: `v3/ploopyco/trackball/trackball_madromys.json`
name in json: `Ploopy Adept Trackball (Madromys)`
?

The main thing with following QMK tree is for easier lookup. 

Also only the name in the JSON actually shows up on VIA website.

Reply:
That suits me just fine. I pushed those changes in ccaca49.